### DISCUSSION TITLE
How to get a section view of 3d results?

### DISCUSSION THREAD

**Discussion by QiangWei-Wade** [GLVis/glvis#129] [Q&A] [answered]
Is there a feature available to get a section view of the domain, e.g. to see what's in there if I cut this beam in half? See illustrated below. I imagine this may not be that difficult given that Glvis can decompose results given by different mpi threads, hopefully?

![section-view](https://user-images.githubusercontent.com/32881293/101906243-9ce0c500-3bf3-11eb-86e0-d4f9c2a169e3.png)

**Comment by tzanio** [comment, marked as answer]
Yes, see the `i` key and the related cutting plane documentation: https://github.com/GLVis/glvis/blob/master/README#L233-L240

**Comment by tzanio** [comment]
Converting from issue to discussion...
